fastcrypto/Cargo.toml

```diff
@@ -22,7 +22,7 @@ serde = { version = "1.0.145", features = ["derive"] }
 serde_bytes = "0.11.7"
 serde_with = "2.0.1"
 signature = { version = "1.6.4", features = ["rand-preview"] }
-tokio = { version = "1.21.2", features = ["sync", "rt", "macros"] }
+tokio = { version = "1.21.1", features = ["sync", "rt", "macros"] }
```

Comment:
sui uses version 1.20.1 which has some features that conflict with 1.21.2, so we need degrade here to avoid a conflict until the version is upgraded in sui.